UniWatch/Controllers/StudentController.cs

```diff
@@ -0,0 +1,191 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Web;
+using System.Web.Mvc;
+using UniWatch.DataAccess;
+using UniWatch.Models;
+using UniWatch.ViewModels;
+
+namespace UniWatch.Controllers
+{
+    public class StudentController : Controller
+    {
+        private IDataAccess _dataAccess;
+
+        public StudentController() : this(new DataAccess.DataAccess())
+        { }
+
+        public StudentController(IDataAccess dataAccess)
+        {
+            _dataAccess = dataAccess;
+        }
+
+        // GET: /Student/Index?classId=1
+        /// <summary>
+        /// Display a list of students in a class
+        /// </summary>
+        /// <param name="classId">The id of the class</param>
+        [HttpGet]
+        public ActionResult Index(int classId)
+        {
+            var @class = _dataAccess.ClassManager.GetById(classId);
+            if (@class == null)
+            {
+                return new HttpStatusCodeResult(HttpStatusCode.BadRequest);
+            }
+
+            var students = _dataAccess.ClassManager.GetEnrolledStudents(classId);
+            return View(new ViewModels.EnrollIndexViewModel
+            {
+                Class = @class,
+                EnrolledStudents = students
+            });
+        }
+
+        /// <summary>
+        /// Returns the view to enroll a student in a class. 
+        /// </summary>
+        /// <param name="classId">The id of the class</param>
+        [HttpGet]
+        public ActionResult Enroll(int classId)
+        {
+            var @class = _dataAccess.ClassManager.GetById(classId);
+            if (@class == null)
+            {
+                return new HttpStatusCodeResult(HttpStatusCode.BadRequest);
+            }
+
+            EnrollStudentViewModel viewModel = new EnrollStudentViewModel
```

Comment:
![image](https://cloud.githubusercontent.com/assets/14304976/20636222/9d2b1404-b32e-11e6-9696-89fb1086af71.png)

Patrick, this is the implementation. It is included in the pull request. It was stored under ViewModels/EnrollmentViewModels.cs